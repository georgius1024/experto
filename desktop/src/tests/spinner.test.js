import React from 'react'
import renderer from 'react-test-renderer'
import Component from '../components/spinner'
describe('Spinner component', () => {
  test('it matches the snapshot', () => {
    const component = renderer.create(<Component />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
