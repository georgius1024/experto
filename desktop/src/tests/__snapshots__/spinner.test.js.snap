// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component it matches the snapshot 1`] = `null`;
